test/system/src/test/java/io/pravega/test/system/AbstractReadWriteTest.java

```diff
@@ -552,6 +573,89 @@ private EventWriterConfig buildWriterConfig() {
         return validEvents;
     }
 
+    void createScope(String scopeName, StreamManager streamManager) {
+        Boolean createScopeStatus = streamManager.createScope(scopeName);
+        if (createScopeStatus) {
+            log.info("Successfully created Scope {}", scopeName);
+        }
+    }
+
+    void createKVT(String scopeName, String kvtName, KeyValueTableConfiguration config, URI controllerURIDirect) {
+        log.info("Creating KVT name:{} in scope {}", kvtName, scopeName);
+        Boolean createKvtStatus = KeyValueTableManager.create(controllerURIDirect).createKeyValueTable(scopeName, kvtName, config);
+        if (createKvtStatus) {
+            log.info("Successfully created KVT {}", kvtName);
+        }
+    }
+
+    void startKVPCreate(String scope, String kvtName, KeyValueTableFactory keyValueTableFactory, URI controllerURIDirect) {
+        testInsertUpdateGetKVP(scope, kvtName, keyValueTableFactory, controllerURIDirect);
+    }
+
+    private void testInsertUpdateGetKVP(String scope, String kvtName, KeyValueTableFactory keyValueTableFactory, URI controllerURIDirect) {
+        log.info("Start KVP operation for KVT {} of scope {}", kvtName, scope);
+        // Creating 5 threads to update KVP table entry
+        for (Integer i = 1; i < 6; i++) {
```

Comment:
This could be an input from the test case, which can decide a different number of writers/readers depending on the test at hand.

Reply:
Resolved, these value provided in as input in test case